src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3878,6 +3885,69 @@ public override IOperation VisitTypeOf(ITypeOfOperation operation, int? captureI
             return new TypeOfExpression(operation.TypeOperand, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
 
+        public override IOperation VisitParameterInitializer(IParameterInitializerOperation operation, int? captureIdForResult)
+        {
+            var parameterRef = new ParameterReferenceExpression(operation.Parameter, semanticModel: null,
+                operation.Syntax, operation.Parameter.Type, constantValue: default, isImplicit: true);
+            VisitInitializer(rewrittenTarget: parameterRef, initializer: operation);
+            return null;
+        }
+
+        public override IOperation VisitFieldInitializer(IFieldInitializerOperation operation, int? captureIdForResult)
+        {
+            foreach (IFieldSymbol fieldSymbol in operation.InitializedFields)
+            {
+                IInstanceReferenceOperation instance = fieldSymbol.IsStatic ?
+                    null :
+                    new InstanceReferenceExpression(InstanceReferenceKind.ContainingTypeInstance, semanticModel: null,
+                        operation.Syntax, fieldSymbol.ContainingType, constantValue: default, isImplicit: true);
+                Optional<object> constantValue = fieldSymbol.HasConstantValue ? new Optional<object>(fieldSymbol.ConstantValue) : default;
+                var fieldRef = new FieldReferenceExpression(fieldSymbol, isDeclaration: false, instance, semanticModel: null,
+                    operation.Syntax, fieldSymbol.Type, constantValue, isImplicit: true);
+                VisitInitializer(rewrittenTarget: fieldRef, initializer: operation);
+            }
+
+            return null;
+        }
+
+        public override IOperation VisitPropertyInitializer(IPropertyInitializerOperation operation, int? captureIdForResult)
+        {
+            foreach (IPropertySymbol propertySymbol in operation.InitializedProperties)
+            {
+                var instance = propertySymbol.IsStatic ?
+                    null :
+                    new InstanceReferenceExpression(InstanceReferenceKind.ContainingTypeInstance, semanticModel: null,
+                        operation.Syntax, propertySymbol.ContainingType, constantValue: default, isImplicit: true);
+                var propertyRef = new PropertyReferenceExpression(propertySymbol, instance, arguments: ImmutableArray<IArgumentOperation>.Empty,
```

Comment:
>ImmutableArray<IArgumentOperation>.Empty [](start = 103, length = 40)

I am curious if there is any way to have an initializer for a property that has parameters, even if that would be an error case.

Reply:
I will try to come up with a unit test, if it is possible.

---
In reply to: [187450406](https://github.com/dotnet/roslyn/pull/26763#discussion_r187450406) [](ancestors = 187450406)

Reply:
I am unable to come up with such a test. Parser either produces an incomplete member for parses it as a field declaration with bracketed argument list.

---
In reply to: [187457859](https://github.com/dotnet/roslyn/pull/26763#discussion_r187457859) [](ancestors = 187457859,187450406)

Reply:
> I am unable to come up with such a test. Parser either produces an incomplete member for parses it as a field declaration with bracketed argument list.

Have tried VB too?

---
In reply to: [187472492](https://github.com/dotnet/roslyn/pull/26763#discussion_r187472492) [](ancestors = 187472492,187457859,187450406)

Reply:
You are correct. VB parses it as a property. Added unit test `NoControlFlow_PropertyInitializerWithArguments`.

---
In reply to: [187475455](https://github.com/dotnet/roslyn/pull/26763#discussion_r187475455) [](ancestors = 187475455,187472492,187457859,187450406)